lighthouse-core/audits/is-on-https.js

```diff
@@ -53,18 +71,16 @@ class HTTPS extends Audit {
           .filter(record => !HTTPS.isSecureRecord(record))
           .map(record => URL.elideDataURI(record.url));
 
+      const items = Array.from(new Set(insecureURLs)).map(url => ({url}));
+
       let displayValue = '';
-      if (insecureURLs.length > 1) {
-        displayValue = `${Util.formatNumber(insecureURLs.length)} insecure requests found`;
-      } else if (insecureURLs.length === 1) {
-        displayValue = `${insecureURLs.length} insecure request found`;
+      if (items.length > 0) {
```

Comment:
I moved this to be `items.length` instead of insecureURLs.length b/c items de-dupes the list and so the displayValue would show "3" when the test for dedupe wanted "2".  Before it was a loose `match` but now it is exact, and made me hunt down why the displayValue differed from the extendedInfo.value and this is what I found.

Reply:
> I moved this to be items.length instead of insecureURLs.length b/c items de-dupes the list 

yes, very good call